avm/res/compute/image/tests/e2e/max/dependencies.bicep

```diff
@@ -185,7 +194,24 @@ resource keyPermissions 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
   }
 }
 
-resource diskEncryptionSet 'Microsoft.Compute/diskEncryptionSets@2022-07-02' = {
+module waitForDeployment 'main.wait.bicep' = {
+  dependsOn: [keyVaultKeyCryptoRole]
+  scope: resourceGroup()
```

Comment:
If this is running in the same scope as everything else in this template - why is in in a seperate file I wonder :)

Reply:
I still wonder @tony-box 😄